### PR TITLE
Bug bash - Fix bugs and refine log in single GPU benchmarks

### PR DESCRIPTION
**Pull request by abuccts**
__Description__

Fix bugs and refine log in single GPU benchmarks.

__Minor Revisions__

- Fix none framework issue.
- Fix empty parameter bug.
- Remove missed mobilenet_v3 models.
- Change benchmark registration log to debug level.
- Add pid in logging.
- Add missing benchmarks in default config.

### CODE REVIEW

File: superbench/executor/executor.py [hunk 1]
```diff
@@ -153,7 +155,7 @@ def exec(self):
             benchmark_config = self._sb_benchmarks[benchmark_name]
             benchmark_results = {}
             self.__create_benchmark_dir(benchmark_name)
-            for framework in benchmark_config.frameworks or [Framework.NONE]:
+            for framework in benchmark_config.frameworks or ['none']:
```
Comment (cp5555) [inline superbench/executor/executor.py:158]: Change to Framework.NONE.value

File: superbench/executor/executor.py [hunk 2]
```diff
@@ -165,7 +167,7 @@ def exec(self):
                             parameters=self.__get_arguments(benchmark_config.parameters)
                         )
                         result = self.__exec_benchmark(context, log_suffix)
-                        if framework != Framework.NONE:
+                        if framework != 'none':
```
Comment (cp5555) [inline superbench/executor/executor.py:170]: Change to Framework.NONE.value

File: superbench/executor/executor.py [hunk 3]
```diff
@@ -179,7 +181,7 @@ def exec(self):
                         parameters=self.__get_arguments(benchmark_config.parameters)
                     )
                     result = self.__exec_benchmark(context, log_suffix)
-                    if framework != Framework.NONE:
+                    if framework != 'none':
```
Comment (cp5555) [inline superbench/executor/executor.py:184]: Change to Framework.NONE.value

**Review by guoshzhao — APPROVED** [review]

**Review by cp5555 — APPROVED** [review]

File: superbench/executor/executor.py (hunk 1 above)
Comment (abuccts) [reply superbench/executor/executor.py]: updated

File: superbench/executor/executor.py (hunk 3 above)
Comment (abuccts) [reply superbench/executor/executor.py]: updated

File: superbench/executor/executor.py (hunk 2 above)
Comment (abuccts) [reply superbench/executor/executor.py]: updated

**Pull request merged**